docs/maintainer/infrastructure.md

```diff
@@ -477,6 +477,34 @@ resource entrusted to conda-forge for use in building packages. We thus cannot s
 feedstocks on any of our CI services. If we find such use, we will politely ask the maintainers to rectify the situation. We may
 take more serious actions, including archiving feedstocks or removing maintainers from the organization, if the situation cannot be rectified.
 
+### External Runners (for GitHub Actions)
+
+conda-forge provides feedstock maintainers a way to contribute/sponsor external runners via [Cirun](#cirun), when
+their CI needs are not fulfilled with the current infrastructure. These external runners (i.e. specialized runners)
+can be used for building one or a bunch of feedstocks, depending on the maintainer. They are runners on a particular
+cloud or a set of clouds, i.e. facilitating provisioning of external runners is basically committing sponsoring of
+provisioning of ephemeral virtual machines on a [supported cloud](https://docs.cirun.io/cloud/). The process for the
+same is described below:
+
+- Share credentials for a supported cloud to a conda-forge admin, so that it can be added to conda-forge's cirun account, alternatively
```

Comment:
```suggestion
- Get in touch with `conda-forge/core` to discuss your use case. This can be done with an issue in the relevant feedstock or via [Zulip](https://conda-forge.zulipchat.com).
- Once discussed, share credentials for a supported cloud with a `conda-forge/core` member, so that it can be added to conda-forge's Cirun account, alternatively
```